onboarding/matplot.py

```diff
@@ -5,3 +5,23 @@
 - Be careful of data points that don't exist
 - Will require reading from CSV, can experiment with pandas
 '''
+
+import pandas as pd
+import matplotlib.pyplot as plt
```

Comment:
You need to add the package imports to `requirements.txt` so that it gets installed with the repo on a `pip install -r requirements.txt`